src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
+
+        System.out.println(ball1.getRandomBall(colorSupplier));
+        System.out.println(ball2.getRandomBall(colorSupplier));
+        System.out.println(ball3.getRandomBall(colorSupplier));
```

Comment:
after you've created Lottery object you should use for loop to create random balls and print them